app/utils/replaceUnlessLoggedIn.js

```diff
@@ -4,11 +4,11 @@ import React, { PureComponent, type ComponentType } from 'react';
 
 type LoginProps = { loggedIn: boolean };
 
-export default function replaceUnlessLoggedIn<Props: Object>(
-  ReplacementComponent: ComponentType<Props>
-): (ActualComponent: ComponentType<Props>) => ComponentType<Props> {
+export default function replaceUnlessLoggedIn(
+  ReplacementComponent: ComponentType<any>
+): (ActualComponent: ComponentType<any>) => ComponentType<any> {
   return (ActualComponent) => {
-    class Replacement extends PureComponent<Props & LoginProps> {
+    class Replacement extends PureComponent<any & LoginProps> {
       render() {
         const { loggedIn, ...props } = this.props;
```

Comment:
Could you leave a comment here that we need to fix this? It shouldn't really be that big of a deal as far as I can see, at least if we get TS. Just something about `replaceUnlessLoggedIn` should have a type parameter with the props of the wrapped component would be nice.

Reply:
Yeah.  Will add comment.

Adding proper typings to it will make all routes using it break... :laughing:

Reply:
Yeah. It definitely is out of scope for this PR 🙃